docs/content/preview/api/ycql/ddl_create_index.md

```diff
@@ -145,6 +146,40 @@ When using an index without transactions enabled, it is the responsibility of th
 
 - A unique index disallows duplicate values from being inserted into the indexed columns. It can be used to ensure uniqueness of index column values.
 
+### DEFERRED INDEX
+
+Currently, an "index backfill" job is launched for each index that is created. For the case where a user creates a table, and adds multiple indexes, this means that the main table will have to be scanned multiple times to populate each index. This is unnecessary, and can also causes issues with the single touch & multi touch block cache algorithm.
+
+After creating a set of indexes with their backfill deferred, the user can then trigger a backfill job for the entire batch of indexes (on the same table) by either:
+
+1. Creating a new index, that is not deferred:
+```cassandraql
```

Comment:
```suggestion

    ```cassandraql
```

Reply:
can you indent the code block after each bullet point by 4 spaces as suggested ?

Reply:
https://github.com/yugabyte/yugabyte-db/pull/20523/commits/c10827917b1c7eaeb133ef8ed977f0183e3f026e